### ISSUE TITLE
nvidia p104-100 problems help

### ISSUE THREAD

**Issue by estiven02** [dartraiden/NVIDIA-patcher#139]
![screeen](https://github.com/user-attachments/assets/e621b46a-ddbb-48fa-bb37-265f106e187b)
good afternoon, could someone help me? The drivers that I installed on my nvidia p104-100 graphics card (8GB vram and 3 fans) do not work or have an error, attached image, it does not tell me how much vram it has, much less is it unlocked for me to use it as a graphics card in video games, I have seen tutorials on YouTube where they use it in video games and how to do it, but those tutorials are very old and the files they used do not appear here to download, I am almost sure that I am making some mistake when configuring the folders (new patch files) and adjustments of these before installing the driver, I will continue investigating but it scares me to think that the graph will be useless, I could use some help, some small well-explained tutorial, or something like that, thank you.

**Comment by dartraiden** [comment]
Reinstall the driver (clean install) with the antivirus disabled, it may have eaten some of the files

**Comment by estiven02** [comment]
> Reinstall the driver (clean install) with the antivirus disabled, it may have eaten some of the files

Hello dartraiden,

I wanted to update you on my situation with the drivers. I am currently using version 551.52, and it detects my GPU as P104-100. I can install them, and indeed, video games work correctly and even better now that I have Atlas OS installed on Windows 10. However, I am facing some persistent issues.

The current drivers work well in games, but I experience a ridiculously strange and severe lag when opening any program that interacts with the GPU hardware, such as Task Manager, MSI Afterburner, and GPU-Z. This behavior makes the experience quite frustrating or at least odd.

Additionally, with the latest driver versions, I have not been able to install them correctly because the last file (InstallCert.nvi) that should open to authorize the drivers is in a strange format that I cannot open like the other two. This has complicated the installation process, and I am unsure how to proceed.

I would greatly appreciate any guidance or solutions you can offer regarding this matter.

Thank you in advance for your help.

If you need any other information or anything else, I will be available.

**Comment by GamerSuper123** [comment]
Здравствуйте у вас после установки ATlas OS только заработало?

**Comment by estiven02** [comment]
> Здравствуйте у вас после установки ATlas OS только заработало?

Yes, Atlas Os works normally, I have not found any bug or anything like that, now the problem is that as I see my graphics is running lower than normal because there is a tutorial on YouTube that tests the nvidia p104 graphics - 100 in games but this person's graphics give 66% better results than mine I made a comparison with the help of gpt chat (artificial intelligence) I have nothing else to attribute this to other than the drivers, that person put the drivers of the gtx 1070 on your graphics but I have not been able to do it... according to gpt chat this could very well be real since the drivers of the gtx 1070 could give it the efficiency and use that this nvidia p104-100 graphics requires if you want to get all the potential, I will continue trying to see what method works to install those drivers to my graphics, the difference between the graphics in the video and mine is that his has 2 fans and mine has 3 (supposed to be better) my graphics never passes of 62 degrees even putting it on the fullmark bechmarck
